### PR TITLE
Restructure directories and imports.

### PR DESCRIPTION
**Pull request by markfickett**

### CODE REVIEW

**Comment by markfickett** [comment]
@dmohs

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router'
 
-import {CohortBuilderComponent} from './cohort-builder.component'
-import {LoginComponent} from './login.component'
-import {SelectRepositoryComponent} from './select-repository.component'
+import {CohortBuilderComponent} from 'app/views/cohort-builder/cohort-builder.component'
```
Comment (dmohs) [inline src/app/app-routing.module.ts]: This naming has some redundancy. Can you just call it `../cohort-builder/component`?

Comment (markfickett) [reply src/app/app-routing.module.ts]: I like it! Seems to work just fine; updated.

**Review by dmohs — APPROVED** [review]
LGTM

**Comment by markfickett** [comment]
Thanks! Please merge.

**Pull request merged**